### PR TITLE
Sulaco QOL and Overhaul.

### PR DESCRIPTION
**Pull request by DerrickChamberlain**
## About The Pull Request

The community wants a Sulaco that flows better. I want a Sulaco that flows better. I overhauled the entire spawn area, and marine prep area. No more individual squad lanes those are now gone.

Before: (Boring ugly soulless, get your feet dirty from the icky toilet water)
Sulacobefore

After: (mind the map stuff I could not figure out how to do a preview)
Sulacoafter2

## Why It's Good For The Game

Community wants ships to flow better, community wants shorter prep times.

## Changelog
:cl:  
qol: Overhauled marine prep, spawn and cafeteria on Sulaco
qol: Added ladders in marine prep that lead outside CIC
qol: Hardened Sulaco AI Core
qol: Hardened Sulaco CIC APC
qol: Moved dropship modules slightly closer on Sulaco
qol: Moved two PACMAN gens to Sulaco Dropship Prep
qol: Added some flavor to the Sulaco (think old marine prep)
del: Removed old marine prep from Sulaco
fix: Returned Req blastdoors on Sulaco
fix: Fixed airlock and shutter buttons appearing through walls
del: Removed a lot of old useless Sulaco areas, and repurposed some
/:cl:

### CODE REVIEW

**Comment by JackTheJackhammer** [comment]
Images are broke

**Comment by JackTheJackhammer** [comment]
Also, can you unfuck the SD? SD is unholdable.

**Comment by DerrickChamberlain** [comment]
> Also, can you unfuck the SD? SD is unholdable.

I have absolutely been thinking about it. I was looking at it somewhat when I was overhauling the spawn area and marine prep.

**Comment by DerrickChamberlain** [comment]
Good atleast I can see the problems now. I never see them on local servers.

**Comment by DerrickChamberlain** [comment]
> Images are broke

Yeah that is why I put this as a draft. Still figuring stuff out. This helps putting it into perspective though.

**Comment by JackTheJackhammer** [comment]
SD could be bigger and the switch could be easier to access. Maybe also just move the entire SD somewhere else. The fact that its so close to the pad is annoying to both xenos and marines.

**Comment by DerrickChamberlain** [comment]
> SD could be bigger and the switch could be easier to access. Maybe also just move the entire SD somewhere else. The fact that its so close to the pad is annoying to both xenos and marines.

I will look into some solutions. Maybe I can find a way to make the area where it is now, better.

**Comment by DerrickChamberlain** [comment]
> Images are broke

Fixed it. Because sometimes I have an unga brain.

**Comment by DerrickChamberlain** [comment]
Ah yes, figured out the problem. Did not see that the vendor tab TM got merged when I was working on this. Easy peasy fix.

**Comment by DerrickChamberlain** [comment]
Also remove the feature tag, code improvement tag please!

**Comment by SlightlyGray** [comment]
+1

File: code/game/area/sulaco.dm [hunk 1]
```diff
@@ -23,6 +23,14 @@
 	name = "Sulaco Bridge Maintenance"
 	icon_state = "maintcentral"
 
+/area/sulaco/security
+	name = "Sulaco Foreship Sec. Office"
```
Comment (XSlayer300) [inline code/game/area/sulaco.dm:27]:
```suggestion
	name = "Sulaco Foreship Security Office"
```

**Comment by TheOneAndMightyRed** [comment]
grafik
red plus is rogue cable
blue line is missing cyan atmos pipe

grafik
Also let me be nitpicky because i don't like missing diagonal corners. ree

**Comment by Alterist** [comment]
> 
> 
> grafik
> red plus is rogue cable
> blue line is missing cyan atmos pipe
> 
> grafik
> Also let me be nitpicky because i don't like missing diagonal corners. ree

this and also ret rid of the 2 wide wall, it always bugged me seeing it on the map

**Comment by DerrickChamberlain** [comment]
> grafik
> red plus is rogue cable
> blue line is missing cyan atmos pipe
> 
> grafik
> Also let me be nitpicky because i don't like missing diagonal corners. ree

The nitpicky bit has always been there, even on the original. I might fix it? Overall I am not to worried about it. 
Thanks for pointing out the missing pipe and rouge cable. I was certain I removed it but sometimes you just forget.
Had a player play as xeno and run along my vent lines, I guess they missed that spot.

**Comment by DerrickChamberlain** [comment]
New weapon room change from the input. It absolutely feels better.
Newweaponroom

File: code/game/area/sulaco.dm (hunk 1 above)
Comment (DerrickChamberlain) [reply code/game/area/sulaco.dm:27]: For the best really.

Comment (DerrickChamberlain) [reply code/game/area/sulaco.dm]: Fixed the typo.

**Comment by DerrickChamberlain** [comment]
Strange. No idea what was throwing error an hour ago. Wish I saw it sooner. Works now!

**Comment by DerrickChamberlain** [comment]
Angry! Fixing my map again.

**Review by TiviPlus — APPROVED** [review]

**Pull request merged**